droidconKE/chai
# Setup lint rules

---
## How can this PR be tested?
```
TBA
```
---

Thank you for opening a Pull Request!
Before submitting your PR, there are a few things you can do to make sure it goes smoothly:


Is this your first Pull Request?


# Screenshot
N/A
![]()
